docs/blocks/block-controls-toolbars-and-inspector.md

```diff
@@ -180,3 +180,160 @@ If you have settings that affects only selected content inside a block (example:
 The inspector region is shown in place of the post settings sidebar when a block is selected.
 
 Similar to rendering a toolbar, if you include an `InspectorControls` element in the return value of your block type's `edit` function, those controls will be shown in the inspector region.
+
+{% codetabs %}
+{% ES5 %}
+```js
+var el = wp.element.createElement,
+	Fragment = wp.element.Fragment
+	registerBlockType = wp.blocks.registerBlockType,
+	RichText = wp.editor.RichText,
+	InspectorControls = wp.editor.InspectorControls,
+	AlignmentToolbar = wp.editor.AlignmentToolbar;
+
+registerBlockType( 'gutenberg-boilerplate-es5/hello-world-step-04', {
+	title: 'Hello World (Step 4)',
+
+	icon: 'universal-access-alt',
+
+	category: 'layout',
+
+	attributes: {
+		content: {
+			type: 'string',
+			source: 'html',
+			selector: 'p',
+		},
+		alignment: {
```

Comment:
Can we use other control? We have a built-in support for alignment so this may be very confusing for developers. It is also included in block controls in core blocks which is the preferred way. Everything else is fine.